extensions/git/package.json

```diff
@@ -1249,17 +1249,17 @@
         },
         {
           "command": "git.revealFileInOS.linux",
-          "when": "scmProvider == git && scmResourceGroup == merge && resourceScheme == file && isLinux",
+          "when": "scmProvider == git && scmResourceGroup == merge && remoteName == '' && isLinux",
```

Comment:
What is this context key trying to prevent, wouldn't remoteName be set in a Remote-WSL window too?

Reply:
These context menu items should only be present if VS Code is not connected to a remote.
The `remoteName` context key is set to `''` when VS Code is not connected to a remote.